src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPipelinePublisher.java

```diff
@@ -268,7 +268,7 @@ public String getStoredPipelineName() {
 
   private JSONArray getPipelineParameters() {
 
-    if (addParam != null && !addParam.isEmpty() && !"{}".equals(addParam)) {
```

Comment:
it seems to be some fix done within the branch

Reply:
That is true. Re-applied fix back.